Update the troubleshooting section (#14805)
DataDog/integrations-core: openmetrics/README.md

Datadog recommends that you use specific metric names or partial metric name matches for more precise collection.

### Errors parsing the OpenMetrics payload with Agent 7.46 and above

> Starting with version 3.0.0 of this integration, that is shipped by default with Agent 7.46 and above, the integration sends by default the `Accept` header set to `application/openmetrics-text;version=1.0.0,application/openmetrics-text;version=0.0.1;q=0.75,text/plain;version=0.0.4;q=0.5,*/*;q=0.1` instead of `text/plain` as it used to do. The integration will then dynamically define which scraper to use based on the `Content-type` it receives from the server.

Reviewer: ```suggestion
Starting with version 3.0.0 of this integration, which is shipped by default with Agent 7.46 and above, the integration sends by default the `Accept` header set to `application/openmetrics-text;version=1.0.0,application/openmetrics-text;version=0.0.1;q=0.75,text/plain;version=0.0.4;q=0.5,*/*;q=0.1`. Previous versions set the `Accept` header to `text/plain`. The integration then dynamically determines which scraper to use based on the `Content-type` it receives from the server.
```

> If you start seeing errors scraping the OpenMetrics endpoint with this new version because the scraper is now stricter, you can manually set the `Accept` header the integration sends to `text/plain` using the `headers` option in the [configuration file][14], for instance: 

Reviewer: ```suggestion
If you see errors scraping the OpenMetrics endpoint with this new version because the scraper is stricter than before, manually set the `Accept` header that the integration sends to `text/plain` using the `headers` option in the [configuration file][14]. For instance: 
```

```yaml
## All options defined here are available to all instances.
#
init_config:
  ...
instances:
  - openmetrics_endpoint: <OPENMETRICS_ENDPOINT>
  ...
    headers:
      - Accept: text/plain
```

> This will cause the endpoint to return a `Content-type` set to `text/plain`, the integration will then use the previous scraper.

Reviewer: ```suggestion
With this configuration, the endpoint returns the `Content-type` set to `text/plain`, causing the integration to use the previous scraper.
```

### Errors parsing the OpenMetrics payload with Agent 7.46 and above

Starting with version 3.0.0 of this integration, which is shipped by default with Agent 7.46 and above, the integration sends by default the `Accept` header set to `application/openmetrics-text;version=1.0.0,application/openmetrics-text;version=0.0.1;q=0.75,text/plain;version=0.0.4;q=0.5,*/*;q=0.1`. Previous versions set the `Accept` header to `text/plain`. The integration then dynamically determines which scraper to use based on the `Content-type` it receives from the server.

If you see errors scraping the OpenMetrics endpoint with this new version because the scraper is stricter than before, manually set the `Accept` header that the integration sends to `text/plain` using the `headers` option in the [configuration file][14]. For instance: 

```yaml
## All options defined here are available to all instances.
#
init_config:
  ...
instances:
  - openmetrics_endpoint: <OPENMETRICS_ENDPOINT>
  ...
    headers:
      - Accept: text/plain
```

With this configuration, the endpoint returns the `Content-type` set to `text/plain`, causing the integration to use the previous scraper.

> The OpenMetrics integration reports an error parsing the payload when the system you are monitoring sends data that does not match their Content-type header. Setting the integration to accept text/plain content creates a workaround to address the problem in the short term.

Reviewer: ```suggestion
The OpenMetrics integration reports an error parsing the payload when the system you are monitoring sends data that does not match their `Content-type` header. Setting the integration to accept `text/plain` content creates a workaround to address the problem in the short term.
```
